lib/msal-common/src/utils/TimeUtils.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @hidden
+ */
+export class TimeUtils {
+    /**
+     * Returns time in seconds for expiration based on string value passed in.
+     *
+     * @param expiresIn
+     */
+    static parseExpiresIn(expiresIn: number): number {
+        // if AAD did not send "expires_in" property, use default expiration of 3599 seconds, for some reason AAD sends 3599 as "expires_in" value instead of 3600
+        if (!expiresIn) {
+            expiresIn = 3599;
+        }
+        return expiresIn;
```

Comment:
Would prefer not to mutate parameters:

`return expiresIn || 3599;`

Reply:
`parseExpiresInSeconds`

Reply:
done.